examples/seq2seq/run_seq2seq.py

```diff
@@ -335,13 +356,17 @@ def main():
     if model.config.decoder_start_token_id is None:
         raise ValueError("Make sure that `config.decoder_start_token_id` is correctly defined")
 
+    # update config with task specific params
+    task_specific_params = model.config.task_specific_params
+    if task_specific_params is not None:
+        params = task_specific_params.get(data_args.task, {})
+        logger.info(f"Updating model.config with task specific params for {data_args.task}:\n {params}")
+        logger.info("Note: command line args may override some of these.")
+        model.config.update(params)
```

Comment:
This was the thing @patrickvonplaten told me to remove, so just pinging him here so you two can fight :-)

Reply:
This is mostly for T5 and for reproducing the metrics. I don't have any strong opinion here. If we decide to remove this, then we should remove all mentions of `task_specific_params` from the script and use the `prefix` only if the user has specified it.

Reply:
I don't care if you want to change this, as long as we can accomplish the same in a new way. 

I have a bit of a hard time understanding what is the intention behind removing functionality. Is this bad functionality? Is it not useful? 

As I mentioned several times in this let's-rewrite-things context, as long as I have a reliable sensitive tool that can help me detect quality regressions over short dataset sample sets I am not attached to any specific way.